packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-items/jurisdiction-modal-foot-note.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { getAuthenticationStatusInfo } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import { TJurisdictionModalFootNoteProps } from '../../props.types';
+
+const JurisdictionModalFootNote = ({
+    account_status,
+    account_type,
+    jurisdiction_selected_shortcode,
+    card_classname,
+}: TJurisdictionModalFootNoteProps) => {
+    const {
+        need_poi_for_vanuatu,
+        need_poi_for_bvi_labuan_maltainvest,
+        need_poa_resubmission,
+        poa_acknowledged,
+        poi_poa_verified_for_bvi_labuan_maltainvest,
+        poi_poa_verified_for_vanuatu,
+        poi_acknowledged_for_bvi_labuan_maltainvest,
+        poi_acknowledged_for_vanuatu,
+        poi_or_poa_not_submitted,
+    } = getAuthenticationStatusInfo(account_status);
+
+    const account_type_name = account_type === 'synthetic' ? 'Synthetics' : 'Financial';
+
+    const is_svg_type = jurisdiction_selected_shortcode && jurisdiction_selected_shortcode === 'svg';
+    const is_vanuatu_type = jurisdiction_selected_shortcode && jurisdiction_selected_shortcode === 'vanuatu';
+    const is_bvi_labuan_maltainvest_type =
+        jurisdiction_selected_shortcode &&
+        (jurisdiction_selected_shortcode === 'bvi' ||
+            jurisdiction_selected_shortcode === 'labuan' ||
+            jurisdiction_selected_shortcode === 'maltainvest');
```

Comment:
Simplify OR checks using Array.includes

Reply:
Thanks @yashim @matin-binary  for the suggestions. I am still refactoring the components as part of new KYC flow. Will incorporate all these suggestions in the new PR